configure.ac

```diff
@@ -3068,7 +3085,7 @@ fi
 # Check for --with-dbmliborder
 AC_MSG_CHECKING(for --with-dbmliborder)
 AC_ARG_WITH(dbmliborder,
-            AS_HELP_STRING([--with-dbmliborder=db1:db2:...], [order to check db backends for dbm. Valid value is a colon separated string with the backend names `ndbm', `gdbm' and `bdb'.]),
+            AS_HELP_STRING([--with-dbmliborder=db1:db2:...], [override order to check db backends for dbm. valid value is a colon separated string with the backend names `ndbm', `gdbm' and `bdb'.]),
```

Comment:
'valid' here is the start of a new sentence, so it should be capitalised.

Reply:
rephrased to avoid starting another sentence